tests/components/wolflink/test_config_flow.py

```diff
@@ -0,0 +1,107 @@
+"""Test the Wolf SmartSet Service config flow."""
+from asynctest import patch
+from httpcore._exceptions import ConnectError
+from wolf_smartset.models import Device
+from wolf_smartset.token_auth import InvalidAuth
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.wolflink.const import DOMAIN
+
+
+async def test_show_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["step_id"] == "user"
+
+
+async def test_device_step_form(hass):
+    """Test we get the second step of config."""
+    conf = {"username": "test-username", "password": "test-password"}
+
+    with patch(
+        "wolf_smartset.wolf_client.WolfClient.fetch_system_list",
```

Comment:
Make sure to patch in the correct place. I'd think that we should patch the library target in the config flow module.

https://docs.python.org/3/library/unittest.mock.html#where-to-patch

Reply:
True we do not need to patch here anymore.

Reply:
At first I didn't understand you comment, after reading link you provided I think I know what you are talking about. Anyway I have change patch in every test.